cypress/integration/repeatedEvents.cy.js

```diff
@@ -81,124 +81,96 @@ describe('repeated events', () => {
         })
         expectHeaderToContainExact(
             0,
-            'Analytics - Percentage - Stage 1 - Repeatable (oldest)'
+            'E2E - Percentage - Stage 1 - Repeatable (oldest)'
         )
         expectHeaderToContainExact(
             1,
-            'Analytics - Percentage - Stage 1 - Repeatable (oldest +1)'
-        )
-        expectHeaderToContainExact(
-            2,
-            'Analytics - Percentage - Stage 1 - Repeatable (oldest +2)'
+            'E2E - Percentage - Stage 1 - Repeatable (oldest +1)'
         )
 
-        // repetition 3/3 can be set successfully
-        setRepetition({ dimensionName, recent: 3, oldest: 3 })
-        result = ['56', '0', '100', '', '5', '']
+        // repetition 2/2 can be set successfully
+        setRepetition({ dimensionName, recent: 2, oldest: 2 })
+        result = ['45', '46', '45', '46']
         result.forEach((value, index) => {
             getTableDataCells().eq(index).invoke('text').should('eq', value)
         })
         expectHeaderToContainExact(
             0,
-            'Analytics - Percentage - Stage 1 - Repeatable (oldest)'
+            'E2E - Percentage - Stage 1 - Repeatable (oldest)'
         )
         expectHeaderToContainExact(
             1,
-            'Analytics - Percentage - Stage 1 - Repeatable (oldest +1)'
+            'E2E - Percentage - Stage 1 - Repeatable (oldest +1)'
         )
         expectHeaderToContainExact(
             2,
-            'Analytics - Percentage - Stage 1 - Repeatable (oldest +2)'
+            'E2E - Percentage - Stage 1 - Repeatable (most recent -1)'
         )
         expectHeaderToContainExact(
             3,
-            'Analytics - Percentage - Stage 1 - Repeatable (most recent -2)'
-        )
-        expectHeaderToContainExact(
-            4,
-            'Analytics - Percentage - Stage 1 - Repeatable (most recent -1)'
-        )
-        expectHeaderToContainExact(
-            5,
-            'Analytics - Percentage - Stage 1 - Repeatable (most recent)'
+            'E2E - Percentage - Stage 1 - Repeatable (most recent)'
         )
     })
     it('repetition out of bounds returns as empty value', () => {
-        const dimensionName = 'Analytics - Percentage'
-        setUpTable({ enrollment: ANALYTICS_PROGRAM, dimensionName })
-
-        // repetition 10/0 can be set successfully
-        setRepetition({ dimensionName, recent: 11, oldest: 0 })
-        const result = [
-            '',
-            '56',
-            '0',
-            '100',
-            '50',
-            '35',
-            '10',
-            '11',
-            '',
-            '5',
-            '',
-        ]
+        const dimensionName = 'E2E - Percentage'
+        setUpTable({ enrollment: E2E_PROGRAM, dimensionName })
+
+        // repetition 6/0 can be set successfully
+        setRepetition({ dimensionName, recent: 6, oldest: 0 })
+        const result = ['', '', '', '', '45', '46']
```

Comment:
previously there was 9 rows and the `10/0` was there to test that the result contained all 9 plus an empty out of bounds result as well. This can't be tested with only 2 rows in the db, so this test now doesn't do its duty..